### PR TITLE
Feat/added storybook

### PR DESCRIPTION
**Pull request by heliomar-pena**
## Explanation

I have added storybook to the project!. You will see in this PR: Some new dependencies, a new script to run storybook, and some news component examples provided by storybook.

I have edited the button example and it's not dinamic (don't change with the params),  but it's because I was testing to use quasar components and SCSS variables!

I think the example's components will be removed soon, I have left it because I haven't time now to add another examples, then will use these for now 😁

## How to test

1. Clone the repo
2. Checkout to this branch
3. `yarn` or `npm i`
4. `yarn storybook`
5. Let me know if it's working!

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -63,36 +36,29 @@ module.exports = {
...
     // in plain CommonJS modules, you can't use `import foo = require('foo')` to pass this rule, so it has to be disabled
     '@typescript-eslint/no-var-requires': 'off',
-
     // The core 'no-unused-vars' rules (in the eslint:recommeded ruleset)
     // does not work with type definitions
     'no-unused-vars': 'off',
-
     // allow debugger during development only
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
```
Comment (Israel-Laguan) [inline .eslintrc.js]: I am shamelessly stealing this rule

File: .eslintrc.js [hunk 2]
```diff
@@ -63,36 +36,29 @@ module.exports = {
...
-    },
-  ],
-};
+  overrides: [{
+    files: ['src/pages/**/*.vue'],
+    rules: {
+      'vue/multi-word-component-names': 0
+    }
+  }]
+};
```
Comment (Israel-Laguan) [inline .eslintrc.js]: Add ELF line please

File: package.json [hunk 3]
```diff
@@ -8,7 +8,9 @@
   "scripts": {
     "lint": "eslint --ext .js,.ts,.vue ./",
     "format": "prettier --write \"**/*.{js,ts,vue,scss,html,md,json}\" --ignore-path .gitignore",
-    "test": "echo \"No test specified\" && exit 0"
+    "test": "echo \"No test specified\" && exit 0",
```
Comment (Israel-Laguan) [inline package.json]: If you don't use jest, I rather prefer to see eslint and prettier here as a sort of "test" if code looks good

**Comment by Israel-Laguan** [comment]
I notice you have yarn and npm, are you supporting both?

**Comment by heliomar-pena** [comment]
@Israel-Laguan 
> I notice you have yarn and npm, are you supporting both?

If using both doesn't cause problems for me is ok.

In case of problems, I prefer to use just Yarn and remove package-lock.json completly

File: package.json (hunk 3 above)
Comment (heliomar-pena) [reply package.json]: Done, it works fine. Thanks for the suggestion!

File: .eslintrc.js (hunk 2 above)
Comment (heliomar-pena) [reply .eslintrc.js]: Thanks!

**Pull request merged**